### ISSUE TITLE
Failure generating documentation with multiple `lightning` libraries

### ISSUE THREAD

**Issue by jkczyz** [rust-bitcoin/rust-lightning#689]
Seeing the following failure after #618 was merged.

```
$ cargo doc
error: The library `lightning` is specified by packages `lightning v0.0.11 (/Users/jkczyz/src/rust-lightning/lightning)` and `lightning-c-bindings v0.0.1 (/Users/jkczyz/src/rust-lightning/lightning-c-bindings)` but can only be documented once. Consider renaming or marking one of the targets as `doc = false`.
```

Workaround is to use `cargo doc -p lightning`.

**Comment by julianknutsen** [comment]
Are docs built as part of CI? I couldn't find it in the build script anywhere but seems like an easy add.

**Comment by TheBlueMatt** [comment]
Ah I never use cargo doc. I just stepped out but I think the fix is just replace the lib name in lightning-c-bindings/Cargo.toml and then update the scripts that specify a binary name. Maybe liblightningbindings or liblightningc or libldk?

**Comment by TheBlueMatt** [comment]
I suppose not :). doctests are run as a part of cargo test, but I guess cargo doc can fail in interesting ways.

**Comment by TheBlueMatt** [comment]
Should be fixed in #688.
